xml/storage_multipath.xml

```diff
@@ -671,389 +658,427 @@
-    <systemitem>DEVICE</systemitem> entry in
-    <filename>/etc/mdadm.conf</filename> should be set as follows:
-   </para>
-<screen>DEVICE /dev/disk/by-id/*</screen>
+   <title>MD RAID on multipathed devices</title>
    <para>
-    If you are using user-friendly names, specify the path as follows so that
-    only the device mapper names are scanned after multipathing is configured:
+     In &productname; 12 and above, MD RAID arrays on top of multipathing
+     are set up automatically by the system's udev rules. No special
+     configuration in <filename>/etc/mdadm.conf</filename> is necessary.
    </para>
-<screen>DEVICE /dev/disk/by-id/dm-uuid-.*-mpath-.*</screen>
   </sect2>
 
-  <sect2 xml:id="sec-multipath-mpiotools-multipath">
-   <title>The multipath command</title>
-   <para>
-    Use the Linux <command>multipath(8)</command> command to configure and
-    manage multipathed devices. The general syntax for the command looks like
-    the following:
-   </para>
-<screen>multipath [-v verbosity_level] [-b bindings_file] [-d] [-h|-l|-ll|-f|-F|-B|-c|-q|-r|-w|-W|-t|-T] [-p failover|multibus|group_by_serial|group_by_prio|group_by_node_name] [<replaceable>DEVICENAME</replaceable>]</screen>
-   <para>
-    Refer to <command>man 8 multipath</command> for details.
-   </para>
-   <bridgehead>General Examples</bridgehead>
+  <sect2 xml:id="sec-multipath-mpiotools-multipathd">
+    <title>The multipathd daemon and command</title>
+    <para>
+      <command>multipathd</command> is the most important part of a modern
+      Linux device-mapper multipath setup. It is normally started through the
+      systemd service <filename>multipathd.service</filename>. Socket activation
+      via <filename>multipathd.socket</filename> is supported, but it's
+      <emphasis>strongly recommended</emphasis> to enable
+      <filename>multipathd.service</filename> on systems with multipath hardware.
+    </para>
+    <para>
+      <command>multipathd</command> serves the following tasks (some of them
+      depend on the configuration):
+    </para>
+      <orderedlist>
+	<listitem>
+	  <para>
+	    On startup, detects path devices and sets up multipath maps from
+	    detected devices, like the <command>multipath</command> command.
+	  </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    Monitors uevents and device mapper events, adding or removing path
+	    devices to multipath maps as necessary, and initiating failover or
+	    failback operations.
+	  </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    Sets up new maps on the fly when new path devices are discovered.
+	  </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    Checks path devices in regular intervals to detect failure, and
+	    tests failed paths to reinstate them if they become operational
+	    again.
+	  </para>
+	</listitem>
+	<listitem>
+	  <para>
+	    When all paths fail, either fail the map, or switch the map
+	    device to queuing mode for a given time interval.
```

Comment:
No this is not "a" queueing mode. There is only one queueing mode. It means that rather than actually submitting I/O to devices, the I/O requests are just queued up  in memory.